use-cases/xtclim/README.md

```diff
@@ -24,14 +24,63 @@ The file `train.py` trains the network. Caution: It will overwrite the weights o
 
 The `anomaly.py` file evaluates the network on the available datasets - train, test, and projection.
 
-## How to launch pipeline
+## Installation
 
-The config file `pipeline.yaml` contains all the steps to execute the workflow. You can launch it from the root of the repository with:
+Please follow the documentation to install the itwinai environment.
+After that, install the required libraries within the itwinai environment with:
 
 ```bash
-python train.py -p pipeline.yaml
+pip install -r Requirements.txt
```

Comment:
to be consistent with the other use cases (and more in general with pip standards), it would be good to rename `Requirements.txt` to `requirements.txt`